javascript-functions/main.js

```diff
@@ -0,0 +1,38 @@
+function convertMinutesToSeconds(minutes) {
+  var timesSixty = minutes * 60;
+  return timesSixty;
+}
+var resultOfConvertMinutestoSeconds = convertMinutesToSeconds(5);
+console.log('resultofConvertMinutestoSeconds:', resultOfConvertMinutestoSeconds);
+
+function greet(name) {
+  var greeting = 'Hey, ' + name;
+  return greeting;
+}
+var resultofGreet = greet('Edwin');
+console.log('resultofGreet:', resultofGreet);
+
+function getArea(width, height) {
+  var widthTimesHeight = width * height;
+  return widthTimesHeight;
+}
+
+var resultOfGetArea = getArea(17, 42);
+console.log('resultOfGetArea:', resultOfGetArea);
+
+function getFirstName(person) {
+  var personFirstName = {
+    firstName: person.firstName
+  };
+  return personFirstName;
```

Comment:
This function isn't meant to return an object, it is meant to return the value of the `firstName` property on the `person` object given as an argument.